### DISCUSSION TITLE
Bluetooth and network management in polybar

### DISCUSSION THREAD

**Discussion by ankeorum** [polybar/polybar#3155] [Q&A]
Hi all,

First of all, thank you so much for this awesome tool you developed! I am trying to build my own customised polybar and I wanted to put next to the sound icon a wifi and bluetooth icons for managing connecting to new networks and connecting new bluetooth devices. Anyone can throw some light?

I have tried enabling the module/network and it detects my wifi through my default device name (wlp0s20f) but clicking on the icon does nothing. Ideally what I would like it to do is to show any other available networks and allowing me connecting to it. 

With the bluetooth icon what I would like it to do is on left-click opening "blueman-manager". If anyone here can let me know of any further steps it would be fantastic.

Apologies if my issues are quite newbie queries but I am not any sort of expert!!

Thank you in advance!

**Comment by parmort** [comment]
You can accomplish the bluetooth module with the `click-left` attribute:

```ini
[module/bluetooth]
type = custom/text
label = YOUR TEXT HERE
click-left = blueman-manager
```

It looks like polybar doesn't allow click attributes for network modules, so you'll need another module like the one above, but for whatever wifi-management program you want to run.
